src/pages-and-resources/discussions/app-list/AppList.jsx

```diff
@@ -37,7 +38,9 @@ function AppList({ intl }) {
   }, [selectedAppId, activeAppId]);
 
   const handleSelectApp = useCallback((appId) => {
-    dispatch(selectApp({ appId }));
+    if (!disabled) {
+      dispatch(selectApp({ appId }));
+    }
```

Comment:
As now it will already be checked by the Card component, we don't need to check here again.